test/Microsoft.Azure.ServiceBus.UnitTests/ExpectedMessagingExceptionTests.cs

```diff
@@ -18,21 +17,28 @@ public class ExpectedMessagingExceptionTests
             var queueClient = QueueClient.CreateFromConnectionString(
                 TestUtility.GetEntityConnectionString(Constants.NonPartitionedQueueName));
 
-            await TestUtility.SendMessagesAsync(queueClient.InnerSender, messageCount);
-            var receivedMessages = await TestUtility.ReceiveMessagesAsync(queueClient.InnerReceiver, messageCount);
+            try
+            {
+                await TestUtility.SendMessagesAsync(queueClient.InnerSender, messageCount);
+                var receivedMessages = await TestUtility.ReceiveMessagesAsync(queueClient.InnerReceiver, messageCount);
 
-            Assert.True(receivedMessages.Count() == messageCount);
+                Assert.True(receivedMessages.Count() == messageCount);
 
-            // Let the messages expire
-            await Task.Delay(TimeSpan.FromMinutes(1));
+                // Let the messages expire
+                await Task.Delay(TimeSpan.FromMinutes(1));
```

Comment:
This is where not having management operations hurts @jtaubensee

A better scenario would be to create an entity with a small lock duration, so that we can continue testing right away.